### PR TITLE
ADD overview.md

### PR DESCRIPTION
**Pull request by andycon**

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+a USB “sniffer” device to capture scanner trigger pulses for use by the capture service, 
+and necessary cables and connectors including audio and video splitter cables. 
+[These details will be filled in later, and there is a reasonable sketch of the hardware set up already in the README.md file.]
+
+The server software runs continuously on a dedicated computer that is properly connected to the stimulus presentation system (SPS) at the fMRI scanner suite. 
+The software monitors the audio and video streams that are sent over the SPS so that whenever there is a video feed to the projector in the scanner, 
+all content is recorded and time-stamped. 
+These recordings are not yet considered “records” in that they are not matched to specific fMRI scans. 
+The server simply monitors the SPS for content and records everything. 
+These raw recordings will often contain long periods of screen capture that show the content of the display on experimenter’s stimulus presentation computer (often a lab laptop).
```
Comment (yarikoptic) [inline overview.md:23]: I think we might want to make it even more explicit like:

```suggestion
These raw recordings will often contain long periods of screen capture that show the content of the display on experimenter’s stimulus presentation computer (often a lab laptop).  
**Warning: as a result, captured video could contain sensitive data if it was displayed.**
```

**Review by yarikoptic — CHANGES_REQUESTED** [review]
Initial pass with some suggestions

File: overview.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+End users, including researchers who collect and analyze data at the center, ideally need not interact with ReproStim in any way except to benefit from having access to stimulus records associated with their fMRI data as procured by the center. 
+Given this general overview of the scope and purpose of the ReproStim project, there are four major components to the project that must be considered. 
+These include hardware requirements and setup, server software configuration, tools for record procurement, and documentation. 
+Details and desiderata for these four components are discussed in turn below.
+
+Setting up ReproStim requires an initial investment in hardware, including a video capture device, a computer that runs the ReproStim capture server software, 
+a USB “sniffer” device to capture scanner trigger pulses for use by the capture service, 
+and necessary cables and connectors including audio and video splitter cables. 
+[These details will be filled in later, and there is a reasonable sketch of the hardware set up already in the README.md file.]
```
[the remark this answers is not in the document]
Comment (yarikoptic) [reply overview.md:15]: will it be done in this PR or you would like to do that in a separate follow up PR?

**Comment by andycon** [comment]
Yarick says to merge this here now.

**Pull request merged**